AsyncDisplayKit/ASTextNode.mm

```diff
@@ -143,6 +148,8 @@ - (instancetype)init
     // on the special placeholder behavior of ASTextNode.
     _placeholderColor = ASDisplayNodeDefaultPlaceholderColor();
     _placeholderInsets = UIEdgeInsetsMake(1.0, 0.0, 1.0, 0.0);
+    
+    _wordKerner = [[ASTextNodeWordKerner alloc] init];
```

Comment:
Can you make this lazy initialize? The init method of text node is already in great need of optimization… of course you should not worry about that right now, but check out ASTextKitRenderer's mechanisms to use getter methods for lazy initialization.  This has the additional benefit of making any dependencies more clear.